etc/config/swift.amazon.properties

```diff
@@ -1,9 +1,9 @@
 compilers=&swift
-demangler=/opt/compiler-explorer/swift-5.9/usr/bin/swift-demangle
-defaultCompiler=swift59
+demangler=/opt/compiler-explorer/swift-5.10/usr/bin/swift-demangle
+defaultCompiler=swift510
 objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 
-group.swift.compilers=swift311:swift402:swift403:swift41:swift411:swift412:swift42:swift50:swift51:swift52:swift53:swift54:swift55:swift56:swift57:swift58:swift59:swift510nightly:swiftnightly
+group.swift.compilers=swift311:swift402:swift403:swift41:swift411:swift412:swift42:swift50:swift51:swift52:swift53:swift54:swift55:swift56:swift57:swift58:swift59:swift510:swiftnightly
```

Comment:
This will break links that pointed to the 5.10 nightly, but seeing that historically we have approved this exact change, I'm happy to approve.

Still, adding a line like
`compiler.swift510.alias=switf510nightly`
would save the old links and make them point to the final 5.10 release :)